### DISCUSSION TITLE
HMR works for you?

### DISCUSSION THREAD

**Discussion by FDiskas** [resend/react-email#1671] [General]
I'm using nextjs and installed `"react-email": "^3.0.1"` everything works fine except that it is not reloading while developing email templates

Looks like it is not tracking changes on custom components

**Comment by FDiskas** [comment]
Also in console I'm having this error https://react.dev/errors/418

**Comment by gabrielmfern** [comment]
Can you share your file structure? If it doesn't work, it might be a bug, since we do have implemented ways to track custom components and imported files.
